microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md

```diff
@@ -0,0 +1,209 @@
+
+See also:
+
+* [Edge Developer Tools for Visual Studio (Preview)](https://devblogs.microsoft.com/visualstudio/?p=237066&preview=1&_ppp=7aa7aef54f), at Visual Studio DevBlogs.
+* [Edge DevTools extension for Visual Studio](../../../../visual-studio/index.md#edge-devtools-extension-for-visual-studio) in _Visual Studio for web development_.
+
+
+<!-- ====================================================================== -->
+## Layers in 3D View
+
+<!-- Title: Layers in 3D View -->
+<!-- Subtitle: The Layers tool isn't going away - find it in the 3D View tool today. -->
+
+We previously added an improved version of Layers in the **3D View** tool, and we're removing the stand-alone version (the **Layers** tool) after users have enough time to make the transition. The **Layers** tool will display a link that redirects you to the **Composited Layers** view within the **3D View** tool.  You can find all the functionality from the **Layers** tool there.
+
+In the following image, the **Focus Mode** UI is shown.  **Focus Mode** has a **Quick View** area at the bottom, in place of the **Drawer**.  The **Layers** tool, which is shown open in **Quick View**, contains only a link, to open the **3D View** tool above it:
+
+![The Layers tool now links to the 3D View tool with the Composited Layers view open.](../../media/2022/03/layers-3d-view-tool.png)
+<!-- work item > layersmove.gif -->
+
+See [Navigate z-index, DOM, and layers using the 3D View tool](../../../3d-view/index.md).
+
+
+<!-- ====================================================================== -->
+## Use your source maps to unminify performance profiles
+
+<!-- Title: The Performance tool can now display unminified call stacks -->
+<!-- Subtitle: Use the new Unminify button in the Performance tool to download an unminified version of the performance profile you recorded. -->
+
+In previous versions of Microsoft Edge, recording a performance profile in the Performance tool would produce a minified call stack. The call stack was not using source maps, even if you hosted them on your server, to unminify the call stack.
+
+Starting in Microsoft Edge version 99, you can unminify a call stack in a performance profile, as follows:
+1. After recording a profile, in the **Performance** tool, click the new **Unminify** button.
+1. Download the unminified performance profile.<!--how?-->
+1. Import the unminified performance profile into the **Performance** tool.<!--how?-->
+
+The **Unminify** button will use source maps, provided that they are hosted side-by-side with your production code, to unminify the call stack in the **Performance** tool.
+
+The minified call stack in the **Performance** tool shows function names as **b**, **O**, **Xt**, and **bn**:
+
+![The minified call stack in the Performance tool.](../../media/2022/03/minified-call-stack-performance-tool.png)
```

Comment:
Please crop the window title bar out of this screenshot. Here's an example of what the image should look like when cropped: 
![image](https://user-images.githubusercontent.com/34316555/157146258-4b383113-0634-485f-abb7-550f678877a1.png)

Reply:
What is the motivation or objective for removing the title bar from these (the Light mode) captures?  If the objective is to not show the URL, I'd rather erase the URL from the title bar rather than erase the entire title bar.  If the objective is to never show title bars, we need to meet to set new conventions for screenshot standards for Win11 screenshots.

Reply:
@zoherghadyali  - Erased URL from the titlebar.  Sufficient?  Or is a requirement that the titlebar be entirely removed?
https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools?branch=pr-en-us-1779#use-your-source-maps-to-unminify-performance-profiles

Reply:
Works for me! Thanks Michael!